### PR TITLE
Add an production ready Dockerfile

### PR DESCRIPTION
**Pull request by florian-vuillemot**

### CODE REVIEW

File: docs/manuals/deployment/docker/production.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+## Prerequisites
+
+- Knowledge of Docker.
+- [Prepare your application for deployment](../prepare-taipy-for-deployment.md)
+- [Followed steps for development](development.md)
+
+
+## Production ready Dockerfile
+
+The following Dockerfile contains minimum configuration for put your application in production.
```
Comment (FabienLelaquais) [inline docs/manuals/deployment/docker/production.md]: "contains minimum configuration for put your application in production"
->
"contains the minimum configuration settings for you to deploy your application to production"

File: docs/manuals/deployment/docker/production.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+This template assumes that you provide a `requirements.txt` file with all your dependencies.
```
Comment (FabienLelaquais) [inline docs/manuals/deployment/docker/production.md]: "your dependencies"
->
"the Python dependencies of your application"

File: docs/manuals/deployment/docker/production.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+Please, update the Docker command `CMD` with corresponding file name and Flask app of your application
```
Comment (FabienLelaquais) [inline docs/manuals/deployment/docker/production.md]: "Please, update the Docker command `CMD` with corresponding file name and Flask app of your application"
->
"Make sure you  update the Docker command `CMD` with the appropriate file name and Flask settings for your application"

File: docs/manuals/deployment/docker/production.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+RUN python -m pip install -r requirements.txt
+
+# Start up command
+ENTRYPOINT [ "gunicorn", "-k", "geventwebsocket.gunicorn.workers.GeventWebSocketWorker", "-w", "1", "--bind=0.0.0.0:5000", "--timeout", "1800" ]
+CMD [ "<main>:<app>" ]
+```
+
+!!! Note
+
+    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to add the following commands
```
Comment (FabienLelaquais) [inline docs/manuals/deployment/docker/production.md]: "using a `SQLDataNode` with a Microsoft SQL Server"
->
"using an `SQLDataNode`  based on Microsoft SQL Server"

**Review by FabienLelaquais — APPROVED** [review]
Minor changes...

File: docs/manuals/deployment/docker/development.md [hunk 5]
```diff
@@ -23,12 +21,12 @@ COPY . /app
...
 
 # Start up command
-CMD python my-app.py
+CMD python main.py
 ```
 
 !!! Note
 
-    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to add the following commands
+    If you are using an `SQLDataNode^` based on Microsoft SQL Server, you need to add the following commands
```
Comment (jrobinAV) [inline docs/manuals/deployment/docker/development.md]: So the doc also works for future sql storage for enterprise.
```suggestion
    If you are using an SQL database based on Microsoft SQL Server, you need to add the following commands
```

What do you think?

File: docs/manuals/deployment/docker/production.md [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+    image: nginx:1.23
+    ports:
+       - "5000:80"
+    volumes:
+      - ./nginx.conf:/etc/nginx/nginx.conf
+```
+
+!!! Warning
+
+    We cannot guarantee that our Dockerfile, Docker Compose and their components do not contain security issues.
```
Comment (jrobinAV) [inline docs/manuals/deployment/docker/production.md]: I would not put this. It is too alarming for me. no?

File: docs/manuals/deployment/linux/redhat.md [hunk 7]
```diff
@@ -46,7 +46,7 @@ sudo restorecon /usr/bin/uwsgi
 
 !!! Note
 
-    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to install your corresponding
+    If you are using a `SQLDataNode^` based on Microsoft SQL Server, you need to install your corresponding
```
Comment (jrobinAV) [inline docs/manuals/deployment/linux/redhat.md]:
```suggestion
    If you are using an SQL database based on Microsoft SQL Server, you need to install your corresponding
```

File: docs/manuals/deployment/linux/ubuntu.md [hunk 8]
```diff
@@ -29,7 +29,7 @@ sudo ln -s `pwd`/.local/bin/uwsgi /usr/bin/uwsgi
 
 !!! Note
 
-    If you are using a `SQLDataNode^` with a Microsoft SQL Server, you need to install your corresponding
+    If you are using a `SQLDataNode^` based on Microsoft SQL Server, you need to install your corresponding
```
Comment (jrobinAV) [inline docs/manuals/deployment/linux/ubuntu.md]:
```suggestion
    If you are using an SQL database based on Microsoft SQL Server, you need to install your corresponding
```

**Review by jrobinAV — APPROVED** [review]

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**